app/forms/provider_interface/confirm_conditions_wizard.rb

```diff
@@ -13,10 +13,15 @@ def initialize(state_store, attrs = {})
     end
 
     def conditions
-      conditions = offer.conditions
-      return conditions if statuses.blank?
+      copied_conditions = offer.conditions.map do |condition|
```

Comment:
can you move the dup block into it's own method as now `conditions` is doing too much?